### ISSUE TITLE
fix windows packing installer alert: unknown publisher

### ISSUE THREAD

**Issue by Ehud-Banunu** [kubernetes/minikube#6735]
**The exact command to reproduce the issue**:
From the listing at https://github.com/kubernetes/minikube/releases ,
on Windows10 Home, attempt to install v1.7.3 or v1.7.2 -
either minikube-windows-amd64.exe or minikube-installer.exe

**The full output of the command that failed**:<details>
<img width="397" alt="Smartscreen_blocks_minikube" src="https://user-images.githubusercontent.com/26869684/75064381-8136f180-54ef-11ea-85f1-9027213c4f12.png">
</details>

**The output of the `minikube logs` command**:<details>
N/A

</details>

**The operating system version**:
Windows10 Home

**Comment by blueelvis** [comment]
Hi!

Thanks for reporting this issue. Could you please click on `More Info`? Once you do, you should see an option to `Run Anyway` like below -
![image](https://user-images.githubusercontent.com/6985763/75090643-decc4c00-558a-11ea-8898-1a2c886f93cd.png)

Kindly click on it and the installation should start. This would work for the installer.

If you just want to use the `minikube-windows-amd64.exe`, please unblock the file by following the procedure over here - https://mywindowshub.com/how-to-unblock-a-file-in-windows-10/

**Comment by afbjorklund** [comment]
This seems to be just as annoying as the Mac Gatekeeper "feature".

So now also the Win binaries needs to be signed (somehow), I guess ?

**Comment by Ehud-Banunu** [comment]
Thanks for the quick response.
@blueelvis I'm familiar with the run-anyway option, but am not at liberty to use it.
Same goes for unblocking the manual-installation.
I'm spinning up Kubernetes manually, as I assume other users facing this issue would.

**Comment by blueelvis** [comment]
@afbjorklund - Yep, this would require the executables to be signed. I have also resorted to using Chocolatey for the installation as it takes care of this.

@Ehud-Banunu - You could try to install minikube using Chocolatey like this - https://minikube.sigs.k8s.io/docs/start/windows/

**Comment by tstromberg** [comment]
#sig-release issue: https://github.com/kubernetes/release/issues/839

**Comment by medyagh** [comment]
how about we add alternative way of installing using something like curl.

**Comment by medyagh** [comment]
@blueelvis do u have suggestions howe we can become a known publisher?

**Comment by tstromberg** [comment]
This issue appears to be a duplicate of #4841, do you mind if we move the conversation there?

Ths way we can centralize the content relating to the issue. If you feel that this issue is not in fact a duplicate, please re-open it using `/reopen`. If you have additional information to share, please add it to the new issue.

Thank you for reporting this!
